src/mame/drivers/nmk16.cpp

```diff
@@ -6540,139 +6516,137 @@ ROM_START( strahljbl ) // N0 892 PCB, this bootleg uses SEIBU sound system
 	ROM_LOAD16_BYTE( "6.ic94",  0x00001, 0x20000, CRC(60d340d0) SHA1(3c6f862901b403d6ddf58823af7d6e3f67573788) )
 
-	ROM_REGION( 0x10000, "audiocpu", 0 )        /* unknown  - sound cpu ?????? */
+	ROM_REGION( 0x10000, "audiocpu", 0 )        // unknown  - sound CPU ??????
 	ROM_LOAD( "1.70",  0x00000, 0x10000, CRC(9e6f48fc) SHA1(aeb5bfecc025b5478f6de874792fc0f7f54932be) )
 
-	ROM_REGION( 0x020000, "fgtile", 0 ) /* Smaller NMK logo plus alternate  Distributed by UPL  Company Limited  starting at tile 0xF80 */
-	ROM_LOAD( "5.ic95",  0x000000, 0x020000, CRC(a2c1e25d) SHA1(cf09cbfd9afc7e3907fef6b26fb269b743f2e036) )   /* 8x8 tiles */
+	ROM_REGION( 0x020000, "fgtile", 0 ) // Smaller NMK logo plus alternate  Distributed by UPL  Company Limited  starting at tile 0xF80
+	ROM_LOAD( "5.ic95",  0x000000, 0x020000, CRC(a2c1e25d) SHA1(cf09cbfd9afc7e3907fef6b26fb269b743f2e036) )   // 8x8 tiles
 
-	ROM_REGION( 0x100000, "bgtile", 0 ) /* 16x16 tiles */
+	ROM_REGION( 0x100000, "bgtile", 0 ) // 16x16 tiles
 	ROM_LOAD( "91076-4.101",  0x000000, 0x100000, CRC(df9653a4) SHA1(4a3204a98d7738c7895169fcece922fdf355f4fa) )
 
 	ROM_REGION( 0x100000, "sprites", 0 )
-	ROM_LOAD16_WORD_SWAP( "91076-8.57",  0x000000, 0x100000, CRC(7fd0f556) SHA1(d1b4bec0946869d3d7bcb870d9ae3bd17395a231) ) /* Sprites */
-
-	ROM_REGION( 0x080000, "oki1", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "91076-2.46",   0x00000, 0x80000, CRC(3f1e67f2) SHA1(413e78587d8a043a0eb94447313ba1b3c5b35be5) ) /* 1st & 2nd half identical, needs verifying */
-	/* 0x20000 - 0x80000 banked */
-
-	ROM_REGION( 0x080000, "oki2", 0 )   /* OKIM6295 samples */
-	ROM_LOAD( "91076-3.45",   0x00000, 0x80000, CRC(b25ed93b) SHA1(d7bc686bbccf982f40420a11158aa8e5dd4207c5) ) /* 1st & 2nd half identical, needs verifying */
-	/* 0x20000 - 0x80000 banked */
+	ROM_LOAD16_WORD_SWAP( "91076-8.57",  0x000000, 0x100000, CRC(7fd0f556) SHA1(d1b4bec0946869d3d7bcb870d9ae3bd17395a231) ) // Sprites
+
+	ROM_REGION( 0x080000, "oki1", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91076-2.46",   0x00000, 0x80000, CRC(3f1e67f2) SHA1(413e78587d8a043a0eb94447313ba1b3c5b35be5) ) // 1st & 2nd half identical, needs verifying
+	// 0x20000 - 0x80000 banked
+	ROM_REGION( 0x080000, "oki2", 0 )   // OKIM6295 samples
+	ROM_LOAD( "91076-3.45",   0x00000, 0x80000, CRC(b25ed93b) SHA1(d7bc686bbccf982f40420a11158aa8e5dd4207c5) ) // 1st & 2nd half identical, needs verifying
+	// 0x20000 - 0x80000 banked
 ROM_END
 
-ROM_START( hachamfb ) /* Thunder Dragon conversion - unprotected prototype or bootleg? */
-	ROM_REGION( 0x40000, "maincpu", 0 )     /* 68000 code */
-	ROM_LOAD16_BYTE( "8.bin",  0x00000, 0x20000, CRC(14845b65) SHA1(5cafd07a8a6f5ccbb36de7a90571f8b33ecf273e) ) /* internally reports as 19th Sep. 1991 */
+ROM_START( hachamfb ) // Thunder Dragon conversion - unprotected prototype or bootleg?
+	ROM_REGION( 0x40000, "maincpu", 0 )     // 68000 Code
```

Comment:
Code is not a proper noun – there’s definitely no reason to be capitalising it here.

Reply:
Fixed, just just here, but also other appearances